File: cppProgramming/Beginner/lcm.cpp
//C++ Program to Find LCM
#include<iostream>
using namespace std;
int main()
{
     int a, b, lcm, ai;
     cout<<"Enter 2 numbers "<<endl;
     cin>>a>>" ">>b;

     for(int i=0;i<=b;i++)
     {
        ai = a*i;
        if(ai%b == 0)
          {
            lcm = ai;
            break;
          }
     }
     cout<<"The LCM of 2 numbers is "<<LCM<<endl;

}

File: pythonProgramming/LCM.py
print("To find the LCM of 2 numbers")
a = int(input("Enter the first number "))
b = int(input("Enter the second number "))
lcm = 0

for x in range(1, b+1):
    ax = a*x
    if ax%b == 0:
        lcm = ax
        break

print("The LCM of the 2 numbers is {}".format(lcm))